protobuf/src/main/java/com/linecorp/armeria/server/protobuf/ProtobufNamedTypeInfoProvider.java

```diff
@@ -129,7 +129,8 @@ private static FieldInfo newFieldInfo(FieldDescriptor fieldDescriptor, Set<Descr
         final TypeSignature typeSignature = newFieldTypeInfo(fieldDescriptor);
         final Object typeDescriptor = typeSignature.namedTypeDescriptor();
         final FieldInfoBuilder builder;
-        if (typeDescriptor instanceof Descriptor && visiting.add((Descriptor) typeDescriptor)) {
+        if (typeDescriptor instanceof Descriptor && visiting.add((Descriptor) typeDescriptor) &&
+            !((Descriptor) typeDescriptor).getFields().isEmpty()) {
```

Comment:
The following validation is pretty annoying and I'm not sure it is genuinely useful validation.
https://github.com/line/armeria/blob/755c28292c734e954bd69aebcfaeabcf7b024c16/core/src/main/java/com/linecorp/armeria/server/docs/FieldInfoBuilder.java#L56-L57

I was wondering if we could solve the problem by just removing the validation.

Reply:
I didn't really take a close look but I guess so 😅